### PR TITLE
Blog post for 0.20.0, change versions

### PR DESCRIPTION
**Pull request by maks-rafalko**

### CODE REVIEW

File: src/_posts/whats-new-in-0.20.0.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+This logger is supposed to be used only with GitHub Actions. It prints GitHub Annotation warnings for *escaped* Mutants right in the Pull Request:
+
+![GitHub Annotation Escaped Mutant](/images/github-logger.png)
+
+Usage (look at [the real example](https://github.com/infection/infection/blob/bef65fc22faa200edd367ffe12596905947a2a93/.github/workflows/mt-annotations.yaml#L50-L52) how Infection uses it itself):
+
+```bash
+# this is needed on GitHub Actions to fetch the base branch to make a diff
+git fetch --depth=1 origin main
```
Comment (sanmai) [inline src/_posts/whats-new-in-0.20.0.md:21]:
```suggestion
git fetch --depth=1 origin $GITHUB_BASE_REF
```

File: src/_posts/whats-new-in-0.20.0.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+Allows filtering files to mutate by using `git diff` with `--diff-filter` option. Sensible values are: `AM` - added and modified files. `A` - only added files.
+
+Best to be used during pull request builds on CI, e.g. with GitHub Actions, Travis CI and so on.
+
+Usage:
+
+```bash
+# this is needed on GitHub Actions to fetch the base branch to make a diff
+git fetch --depth=1 origin main
```
Comment (sanmai) [inline src/_posts/whats-new-in-0.20.0.md:48]:
```suggestion
git fetch --depth=1 origin $GITHUB_BASE_REF
```

File: src/_posts/whats-new-in-0.20.0.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+### `--git-diff-base` option
+
+Supposed to be used only with [`--git-diff-filter`](/guide/command-line-options.html#git-diff-filter) option. Configures the base branch for `git diff` command.
+
+Usage:
+
+```bash
+# this is needed on GitHub Actions to fetch the base branch to make a diff
+git fetch --depth=1 origin main
```
Comment (sanmai) [inline src/_posts/whats-new-in-0.20.0.md:65]:
```suggestion
git fetch --depth=1 origin $GITHUB_BASE_REF
```

File: src/_posts/whats-new-in-0.20.0.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+infection.phar --git-diff-base=origin/main --git-diff-filter=AM
```
Comment (sanmai) [inline src/_posts/whats-new-in-0.20.0.md:67]:
```suggestion
infection.phar --git-diff-base=origin/$GITHUB_BASE_REF --git-diff-filter=AM
```

**Review by sanmai — APPROVED** [review]
...If we assume that examples should be usable from the get-go.

**Comment by maks-rafalko** [comment]
Thank you @sanmai

**Pull request merged**